docs/topics/asyncio.rst

```diff
@@ -98,6 +98,52 @@ Futures. Scrapy provides two helpers for this:
          into your own code.
 
 
+Async additional requests
+=========================
+
+The spider below shows a single use-case of scraping a page and gathering a price from a separate URL::
+
```

Comment:
```suggestion
The spider below shows how to send a request and await its response all from 
within a spider callback:

.. code-block:: python
```

PS: Using `.. code-block:: python` will trigger a pre-commit error until you apply black formatting to the code.

Reply:
Duplicated.